### PR TITLE
Don't patch openssl by default and make --sulong implicit

### PR DESCRIPTION
**Pull request by chrisseaton**

### CODE REVIEW

File: ci.hocon [hunk 1]
```diff
@@ -423,8 +423,16 @@ test-gems: ${gem-test-pack} {
   ]
 }
 
-test-ecosystem: ${gem-test-pack} {
-  setup: ${common.setup} ${gem-test-pack.setup}
+test-bundle: ${sulong} {
+  setup: ${common.setup} ${sulong.setup}
+  
+  run: [
+    ${jt} [test, bundle]
```
Comment (eregon) [inline ci.hocon]: `jt test bundle` is already run with Sulong as part of `test-cexts`, maybe we want to keep testing the variant without Sulong here?

Comment (eregon) [inline ci.hocon]: with/without Sulong => without/with OpenSSL patching rather

Comment (chrisseaton) [inline ci.hocon]: Good idea.

**Comment by chrisseaton** [comment]
This isn't going to be as simple as I thought. All the ecosystem commands need to be run with `--sulong`, but there's several sub-rubies involved I believe, run by the tool.

Maybe we should make `--sulong` implicit if there is a Sulong sibling repository?

**Comment by eregon** [comment]
Yes, I think it's reasonable to add the two Sulong flags automatically in the Bash launcher if there is `../sulong/build/sulong.jar`.

**Comment by chrisseaton** [comment]
@eregon everything relevant passes now - can you do a final review.

File: bin/truffleruby.sh [hunk 2]
```diff
@@ -114,6 +114,12 @@ if [ $on_graalvm = false ]; then
     CP="$CP:$root/mxbuild/dists/truffleruby-launcher.jar"
     CP="$CP:$root/mxbuild/dists/truffleruby.jar"
     java_args+=("-Dtruffle.nfi.library=$truffle/truffle-nfi-native/bin/libtrufflenfi.$(libext)")
+    sulong_root="$root/../sulong"
```
Comment (eregon) [inline bin/truffleruby.sh]: Could you use `$root_parent` instead of `$root/..` here?

Comment (chrisseaton) [inline bin/truffleruby.sh]: Done.

**Pull request merged**